matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/homeserver/CapabilitiesAPI.kt

```diff
@@ -27,4 +27,10 @@ internal interface CapabilitiesAPI {
      */
     @GET(NetworkConstants.URI_API_MEDIA_PREFIX_PATH_R0 + "config")
     fun getUploadCapabilities(): Call<GetUploadCapabilitiesResult>
+
+    /**
+     * Request the versions
+     */
+    @GET(NetworkConstants.URI_API_PREFIX_PATH_ + "versions")
+    fun getVersions(): Call<Unit>
```

Comment:
This is already define in AuthAPI, which cannot be injected. My concern is more that this API does not need any access token, and here we will provide it. We should avoid to do that.
We should create a new API file, maybe called UnauthenticatedAPI

Reply:
Or rather add a annotation to avoid adding the token.
Other API are in the same case (ex: publicRooms), so this is not a new pb. Maybe do not do anything and we will do it in another PR?

Reply:
Check with the Synapse team, and they do not think it's an issue